### PR TITLE
fix addDominant test

### PR DESCRIPTION
**Pull request by theotimeee**
This PR aims to fix the following failing test:
```
── Failure ('test-5dominantTaxa.R:44:9'): getDominant ──────────────────────────
colData(addDominant(tse, rank = "Genus", na.rm = FALSE, name = "dominant"))$dominant[1:15] not equal to `exp.vals.two`.
8/15 mismatches
x[1]: "Class:Thermoprotei"
y[1]: "Genus:CandidatusSolibacter"

x[2]: "Class:Thermoprotei"
y[2]: "Genus:MC18"

x[3]: "Class:Thermoprotei"
y[3]: "Class:Chloracidobacteria"

x[7]: "Class:Thermoprotei"
y[7]: "Family:Moraxellaceae"

x[12]: "Class:Thermoprotei"
y[12]: "Family:ACK-M1"
```
The issue was that the default value of `onRankOnly` in `agglomerateByRank` is `TRUE` so `addDominant` needed to be explicitly called with `onRankOnly = FALSE` as it was done for `getDominant`.

### CODE REVIEW

**Comment by TuomasBorman** [comment]
Alright, thanks for checking this. So this PR https://github.com/microbiome/mia/pull/556/files should already fix the problem. Let's wait until the PR is merged and check.

**Review by TuomasBorman — CHANGES_REQUESTED** [review]
Wait until the https://github.com/microbiome/mia/pull/556/files is merged

**Comment by TuomasBorman** [comment]
Can you check whether this change is needed anymore?

**Comment by theotimeee** [comment]
> Can you check whether this change is needed anymore?

This change is not needed anymore, the test is not failing anymore on devel branch.

**Comment by TuomasBorman** [comment]
Thanks, closing

**Pull request closed without merging**
